third_party/robin-map/tsl/robin_hash.h

```diff
@@ -1116,11 +1116,26 @@ class robin_hash: private Hash, private KeyEqual, private GrowthPolicy {
     void insert_value(std::size_t ibucket, distance_type dist_from_ideal_bucket, 
                       truncated_hash_type hash, Args&&... value_type_args) 
     {
-        insert_value(ibucket, dist_from_ideal_bucket, hash, value_type(std::forward<Args>(value_type_args)...));
+        value_type value(std::forward<Args>(value_type_args)...);
```

Comment:
Is this an update of robin-map from upstream or changes you made yourself? If the latter, we should upstream those changes or at least track them in a README somewhere otherwise we'll lose them.

Reply:
This is aa fix that was made in the robin_map github directory. I added the url of the change in a comment.

Reply:
We should probably update `robin-map` entirely instead of taking single changes like this. I believe we have a .txt in the robin-map directory that explains how to do this.

Reply:
Well, the change is pretty specific got a msvc issue and is not dependent on other changes. I was thinking that if you guys want to pull the latest version of robin-map, it would probably be better for you guys to do it. That being said I can do it if you wish (I don't see the .txt file you refer to in robin-map directory).

Reply:
I'll update `robin-map` to the latest master

Reply:
great, thanks @bejado. I'll remove the robin-map change from my PR then.

Reply:
@bejado, I did remove my change in `robin-map`.

Reply:
So the latest `robin-map` does not appear to be compatible with Visual Studio 2015 headers. It works with VS 2017, but given that our CI only has 2015, I think we should proceed with this single cherry-picked change. @romainguy what do you think?

Reply:
I'll be happy to add it back, I agree it seems like the best solution at this point.

Reply:
@bejado, I added the cherry-picked change to `robin-map` back